frontend/src/pages/modelRegistry/screens/ModelVersionDetails/ModelVersionRegisteredDeploymentsView.tsx

```diff
@@ -35,6 +37,17 @@ const ModelVersionRegisteredDeploymentsView: React.FC<
     );
   }
 
+  const getColumnsWithKebab = (projects: ProjectKind[]) => {
+    const columns = getVersionDetailsInferenceServiceColumns(projects);
+    return [
+      ...columns,
+      {
+        ...COL_KEBAB,
+        actions: [],
+      },
+    ];
+  };
+
```

Comment:
We don't need this, we can instead just modify `getVersionDetailsInferenceServiceColumns` to add `COL_KEBAB` since this is the only place it's used / it is specifically for this view.

Reply:
@mturley Done :)